src/quacc/recipes/common/phonons.py

```diff
@@ -89,17 +124,25 @@ def _get_forces_subflow(supercells: list[Atoms]) -> list[dict]:
     @job
     def _thermo_job(
         atoms: Atoms,
-        phonopy: Phonopy,
+        phonopy,
         force_job_results: list[dict],
-        t_step: float,
-        t_min: float,
-        t_max: float,
-        additional_fields: dict[str, Any] | None,
+        t_step,
+        t_min,
+        t_max,
+        additional_fields,
```

Comment:
I assume you just removed the type hints for brevity's sake since they are basically implied elsewhere? I was also thinking about this since it is an internal function. Although, if we at some point enabled static type checking, having the type hints would ensure there isn't an error. I doubt we will be able to get to the point where we enable static type checking though... certainly not anytime soon.